exodus_lambda/functions/origin_request.py

```diff
@@ -122,7 +122,7 @@ def secret(self):
                 self._cache["secret"] = out
                 self.logger.info("Loaded and cached secret %s", arn)
             except Exception as exc_info:
-                self.logger.error(
+                self.logger.exception(
                     "Couldn't load secret %s", arn, exc_info=exc_info
                 )
                 raise exc_info
```

Comment:
Minor: the main point of logger.exception() is that it automatically passes exc_info of the current exception to the logger.  So this code is more complicated than it needs to be.

I believe it will work, but it would be more idiomatically written as:

```
except Exception:
    self.logger.exception("Couldn't load secret %s", arn)
    raise
```

No need to catch the exception into a variable and no need to pass exc_info argument.

Reply:
Thanks. I'll clean up the other logger.exception calls as well.